package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "tslint-config-shopify",
+  "version": "1.0.0",
+  "description": "Shopify's baseline ESLint config.",
+  "keywords": [
+    "tslint",
+    "tslintconfig",
+    "tslintplugin",
+    "tslint-config",
+    "tslint-config-shopify",
+    "shopify"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "tslint": "tslint -c index.js *.ts --project tsconfig.json --type-check",
+    "eslint": "eslint . --max-warnings 0"
```

Comment:
Linting TSLint rules with ESLint so that other people can TSLint using our ESLint-validated rules :mindblown: